Application/EdFi.Ods.AdminApi/Program.cs

```diff
@@ -23,6 +26,13 @@
 builder.Services.AddSingleton<IRateLimitConfiguration, RateLimitConfiguration>();
 builder.Services.AddInMemoryRateLimiting();
 
+builder.Services.Configure<AdminConsoleSettings>(builder.Configuration.GetSection("AdminConsoleSettings"));

```

Comment:
the code from line 29 to line 35 has to be moved to our static class for service registration like in line 39: 
`ServiceRegistration.AddServices(builder.Services, builder.Configuration);`

because this has to be enabled only when the "EnableAdminConsoleAPI" flag is in true.

Also, the "AdminConsoleSettings" section is duplicated because the appsettings has already have it but with different, you will find it like this (It's up to you if you want to rename it):

```
"AdminConsole": {
    "CorsSettings": {
        "EnableCors": false,
        "AllowedOrigins": [
            "https://localhost"
        ]
    }
} 
```